source/languageDetection.py

```diff
@@ -0,0 +1,284 @@
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2015 Dinesh Kaushal, NV Access Limited
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+
+
+"""Handles detection of languages in Unicode text.
+"""
+
+import speech
+from speech import SpeechCommand
+from speech import LangChangeCommand
+import languageHandler
+import config
+from unicodeScriptData import scriptCode
+from collections import OrderedDict
+from logHandler import log
+import unicodedata
+
+# maintains list of priority languages as a list of languageID, ScriptName, and LanguageDescription
+languagePriorityListSpec = []
```

Comment:
This might be a good place to use a `namedtuple`, then we can refer to these by name rather than remembering the order.

Reply:
Thanks for your suggestions! using namedtuple has improved code readability. I didn't know about this feature.